src/compat/process.c

```diff
@@ -186,11 +187,22 @@ int resume_child(void) {
         exit(-1);
 
     local_ctx = *ctx;
+    struct test_single_param *param = NULL;
+    if (local_ctx.param.size != 0) {
+        param = malloc(sizeof (struct test_single_param) + local_ctx.param->size);
+        *param = (struct test_single_param) {
+            .size = local_ctx.param->size,
+            .ptr = param + 1,
+        };
+        memcpy(param + 1, local_ctx.param->ptr, param->size);
```

Comment:
Seems like these should be `local_ctx.param.ptr`, `local_ctx.param.size` etc. according to MinGW and MSVC. Not sure how it is building with GCC! strange. :smile: